lib/insights/api/common/metrics.rb

```diff
@@ -51,6 +53,10 @@ def self.activate(config, prefix, args = {})
             end
           end
         end
+
+        private_class_method def self.metrics_port
+          @metrics_port ||= (ENV['METRICS_PORT']&.to_i || 9394)
```

Comment:
@lindgrenj6 Change looks good, just wanted to ask if we feel comfortable with the ENV var name.  It is short and generic so not sure if we would run into an issue anywhere.

My only other name suggestion would be `INSIGHTS_METRICS_PORT`.  But if others don't see an issue (because it is only for local test envs) then I think it is fine.

Reply:
The main reason for the name is because all of the other services that export prometheus metrics use the same `ENV` var, I'd be down for changing it to something else like `INSIGHTS_METRIC_PORT`/`PROMETHEUS_PORT` since that's even more descriptive, but I think I'd want to change it everywhere to be consistent.

Reply:
Good to know.  Makes sense to keep the current value to be consistent.  👍